purity/gc/gc.go

```diff
@@ -543,7 +546,7 @@ func (c *collector) RemoveBlobs() error {
 		time.Sleep(10 * time.Millisecond)
 	}()
 	var wg sync.WaitGroup
-	wg.Add(numBlobs)
```

Comment:
At that point, I think you can get rid of `numBlobs`, as it's not used anywhere.. I think?

Reply:
We use it in the notice saying how many blobs we are going to delete. I think it's worth keeping